flake.nix

```diff
@@ -0,0 +1,39 @@
+{
+  # Override nixpkgs to use the latest set of node packages
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/master";
```

Comment:
We probably don't want `master` here.
Maybe `nixpkgs-unstable` or `nixos-unstable` if we want new-ish packages.

Reply:
-> go_1_20 only exist there